src/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```

Comment:
Since you are creating `src/index.html`, don't you also want to delete `index.html` from the project's root directory?

Reply:
Yes, I forgot it.